lib/stylesheet/manager/builder.rb

```diff
@@ -0,0 +1,274 @@
+# frozen_string_literal: true
+
+class Stylesheet::Manager::Builder
```

Comment:
This used to be `Stylesheet::Manager` but I need an instance of `Stylesheet::Manager` per request so moving this to another class as a builder. Copied and pasted instance methods from `Stylesheet::Manager` with some slight tweaks.